src/core/fonts.js

```diff
@@ -5326,15 +5326,21 @@ var Type1Parser = (function Type1ParserClosure() {
   }
 
   function decrypt(data, key, discardNumber) {
-    var r = key | 0, c1 = 52845, c2 = 22719;
-    var count = data.length;
+    if (discardNumber >= data.length) {
+      return new Uint8Array(0);
+    }
+    var r = key | 0, c1 = 52845, c2 = 22719, i, j;
+    for (i = 0; i < discardNumber; i++) {
+      r = ((data[i] + r) * c1 + c2) & ((1 << 16) - 1);
```

Comment:
Talked with one of the spider monkey people, the value will be constant folded by the parser, so it doesn't matter.